scala/scala.bzl

```diff
@@ -732,7 +732,7 @@ scala_repl = rule(
 
 def scala_version():
   """return the scala version for use in maven coordinates"""
-  return "2.11"
+  return "2.12"
```

Comment:
wouldn't this be a problem? Don't we need to solve #80 (cross building) before merging this PR?

Reply:
I'd probably go with the 2.12-specific branch for now and consolidate both when #80 is solved.

Reply:
I see. Main issue to consider is that we're in a few months of  adding a host of new features to master so you'll need to catch up if we ever want to merge it in.

Reply:
A branch is okay, but I share ittaiz anxiety about a long lived branch. I hope we can very quickly get the version configurable.